pio/src/iSpindel.cpp

```diff
@@ -335,20 +336,20 @@ bool startConfiguration()
   WiFiManagerParameter custom_api("selAPI", "selAPI", String(my_api).c_str(),
                                   20, TYPE_HIDDEN, WFM_NO_LABEL);
 
-  WiFiManagerParameter custom_name("name", "iSpindel Name", htmlencode(my_name).c_str(),
+  WiFiManagerParameter custom_name("name", "iSpindel Name (AWS ThingName)", htmlencode(my_name).c_str(),
```

Comment:
AWSIOT should not be that prominent as it would be the suggested endpoint.

In fact rather I'd put a warning that the SSL & NTP will add a time penalty which will negatively impact battery lifetime

Reply:
Hi, I'm happy to make those changes. How do I do it? Do I change on my branch and submit a new pull request? Or is there a way to do here?

One note though, I've spent quite a bit of time and have been unable to replace secrets.h wih input parameters on the WIFI Manager. Is this viable base code if users have to edit a secrets file and compile for themselves? (Or any suggestions on how to capture these through WiFiManager?)

Reply:
@camsaway if you push a new commit to your branch (`AWS`) it will show up in this pull request

Reply:
Thanks Linus. Will let you know when tested.

Reply:
Hi @universam1 - I have made changes as requested. I removed reference to AWS from the main GUI and if you select AWSIOTMQTT as the API then you get a warning message about battery usage as well as the "AWS Hints".
I also added a Readme_AWS.md file